Source/Core/Core/IOS/Network/KD/NWC24DL.h

```diff
@@ -44,11 +42,16 @@ class NWC24Dl final
   void SetVersion(u32 version);
 
 private:
-  enum
+  static constexpr u32 DL_LIST_MAGIC = 'WcDl';
+  static constexpr u32 MAX_SUBENTRIES = 32;
+  static constexpr u32 MAX_ENTRIES = 120;
+
+  enum EntryType : u8
   {
-    DL_LIST_MAGIC = 'WcDl',  // 'WcDl'
-    MAX_SUBENTRIES = 0x20,   // 32
-    MAX_ENTRIES = 0x78,      // 120
+    unk = 1,
+    mail,
+    channel_content,
+    unused = 0xff
```

Comment:
We generally have enum names in CamelCase, unless there's a specific reason you have them like this?

Reply:
IIRC, `enum`s tend to follow the style for compile-time constant variables (i.e. fully uppercased) while `enum class` tend to have CamelCase. Either way, we don't use snake_case for enum.